source/code/plugins/out_oms_changetracking_file.rb

```diff
@@ -113,7 +113,10 @@ def start
 
          @log.debug "contentlocationfilepath : #{contentlocationfilepath}"
          if File.exists?(contentlocationfilepath)
-            @@LastContentlocationUri = File.open(contentlocationfilepath, &:gets).strip
+            content = File.open(contentlocationfilepath, &:gets)
+            if !content.nil? and !content.empty?
+               @@LastContentlocationUri = content.strip
```

Comment:
I would also check LastContentlocatioUri agains "nil" and "empty. Also rename it to LastContentLocationUri. L with upper case.